Kombit.Samples.CH.WebsiteDemo/MyPage.aspx.cs

```diff
@@ -75,7 +76,11 @@ protected void Btn_ReloginNoForceAuthnAssuranceLevel4_Click(object sender, Event
 
         private void HandleReloginNoForceAuthnAssuranceLevel(int assuranceLevel)
         {
-            SessionFactory.SessionContext.Current[SessionConstants.RequestedAssuranceLevel] = "urn:dk:gov:saml:attribute:AssuranceLevel:" + assuranceLevel;
+            var session = SessionStore.CurrentSession;
+            if (session != null)
+            {
+                SessionStore.CurrentSession[SessionConstants.RequestedAssuranceLevel] = "urn:dk:gov:saml:attribute:AssuranceLevel:" + assuranceLevel;
```

Comment:
This is not the right OIOSAML3 claim type. You can find the right one in the list that NGT will give you